Content/default/src/Client/Index.fs

```diff
@@ -34,83 +34,64 @@ let update (msg: Msg) (model: Model) : Model * Cmd<Msg> =
         let cmd = Cmd.OfAsync.perform todosApi.addTodo todo AddedTodo
 
         { model with Input = "" }, cmd
-    | AddedTodo todo -> { model with Todos = model.Todos @ [ todo ] }, Cmd.none
+    | AddedTodo todo ->
+        { model with
+            Todos = model.Todos @ [ todo ] },
+        Cmd.none
 
 open Feliz
 
 let private todoAction (model: Model) (dispatch: Dispatch<Msg>) =
-    Html.div [
-        prop.className "flex mt-4"
-        prop.children [
-            Html.input [
-                prop.className "shadow appearance-none border rounded w-full py-2 px-3 mr-4 text-grey-darker"
-                prop.value model.Input
-                prop.placeholder "What needs to be done?"
-                prop.autoFocus true
-                prop.onChange (SetInput >> dispatch)
-                prop.onKeyPress (fun ev ->
-                    if ev.key = "Enter" then
-                        dispatch AddTodo)
-            ]
-            Html.button [
-                prop.className
-                    "flex-no-shrink p-2 rounded bg-teal-300 hover:bg-teal disabled:opacity-30     disabled:cursor-not-allowed"
-                prop.disabled (Todo.isValid model.Input |> not)
-                prop.onClick (fun _ -> dispatch AddTodo)
-                prop.text "Add"
-            ]
-        ]
-    ]
+    Html.div
```

Comment:
This isn't the intended effect of the new setting. I think the setting name is now `stroustrup` rather than `experimental_stroustrup`

Reply:
Good catch for the setting.

I did re-run `dotnet run Format` after fixing the setting because Fantomas didn't touch the formatting.

Reply:
It seems that it still hasn't corrected the formatting in all files. I'm not sure what's going on there. Maybe we are better off without the setting after all 🤔

Reply:
Changing the setting and re-running `dotnet run Format` indeed formatted no files which feels a bit strange.

I can revisit again if needed, but the templates / files are probably going to be touch again I was not too concerned about it yet.